docs/security/security.md

```diff
@@ -0,0 +1,120 @@
+
+Here are some security recommendations to help the team minimize any possible attack vectors. This list is by no means an exhaustive list of guidelines and will evolve in the future.
+
+## Multi factor authentication(MFA)
+
+We should ensure that MFA is used at all times, limiting any attack surface substantially.
+
+## Passwords
+
+As a team, we do not enforce a password schema, but passwords should be as secure as you deem so; some general guidelines are:
+
+- different from other passwords
+- safe from external viewing
+- no obvious personal information like your name
+
+## Data
+
+### Sensitive Data
+
+Sensitive data is anything that might risk the company if an external party sees it. This data could be:
+- environment variables like secrets, keys, passwords, endpoints, tokens
+- customer/partner details
+- company assets
+- projects
+
+If any company asset is downloaded to a local machine, it should serve its purpose and be deleted from the local machine or encrypted.
+
+### At rest
+
+[Data at rest][data-at-rest] is data when it is not moving. Such as:
+- stored on some device
+- on some cloud platform
+- in a database
+- in your browser
+
+There are many risks involved with data at rest:
+- physical, such as burglary, robbery, theft
+- digital, such as device compromise
+
+In general, the key to mitigating this risk is to ensure that everything is encrypted and unnecessary data is removed when it is no longer needed.
+
+The best-in-class solution for encrypting data at rest is to encrypt the entire drive using something like [LUKS][luks]. This is not always feasible so you can use something like [EcryptFS][ecryptfs] to encrypt individual folders/files. These tools are not exhaustive, there are variants in all operating systems.
+
+Some great tools:
+- [GPG][gpg]
+- [EcryptFS][ecryptfs]
+- [LUKS][luks]
+
+### In transit
+
+[Data in transit][data-in-transit] is any time data is *moving* from one place to another.
+
+Careful consideration should be taken about the data is sent across:
+- insecure networks, such as HTTP
+- chat apps like slack, discord, telegram
+- open networks like coffee shops, restaurants
+
+A note on chat apps:
+
+While not generally insecure, chat apps still open an unnecessary risk of leaked data and open an attack vector of the respective companies' access to our data.
+
+## Logging
+
+Logging is an essential tool in the Development Process. 
+
+It does, however, carry some risk; Developers should take care to ensure that no sensitive data is logged. And any logs are redacted if passed on an insecure channel.
+
+## Git usage
+
+As a team, we need to be mindful of what we commit to the repository, that is, to ensure no secrets or sensitive data gets leaked in a commit. 
+If this happens, you must ensure the data is not leaked. Usually, this would be rectifying the leakage and force pushing.
+
+To mitigate this, always make sure you inspect your commits before pushing.
+
+### Commit Verification
```

Comment:
We should enforce this at the org level, @darkfriend77 if you get time, would you mind doing that? That way all new repos will have it enforced automatically.

Reply:
Only signed and verified commits, let me check where I can enforce that.